apps/web/vibes/soul/sections/error/index.tsx

```diff
@@ -24,10 +21,12 @@ export function Error({
         </h1>
         <p className="text-lg text-contrast-500">{subtitle}</p>
 
-        {cta !== undefined && cta.href !== '' && cta.label !== '' && (
-          <ButtonLink className="mt-8" href={cta.href} size="large" type="button" variant="primary">
-            {cta.label}
-          </ButtonLink>
+        {ctaAction && (
+          <form action={ctaAction}>
+            <Button className="mt-8" size="large" type="submit" variant="primary">
+              {ctaLabel}
+            </Button>
+          </form>
```

Comment:
Do we need a form? Per the [docs](
@bigcommerce/catalyst-core:dev: [BigCommerce] query GetFavicon - 84ms - complexity 1002
) a simple `onClick` handler should work.

Reply:
@jorgemoya We don't need a form since NextJS error pages are always client components, but @migueloller suggested using a form for versatility - just in-case someone might want to use the component in a non-standard setting